src/index.ts

```diff
@@ -325,8 +332,6 @@ function startSpinning() {
 const button = document.getElementById('startSpinning');
```

Comment:
It'd be great if you could rename this one to something more suitable

Reply:
Done and done! I just realized I don't need to keep the commit names matching the issue name and should actually describe my commit lol. Sorry about that!

Reply:
Github is not showing any changes to this button variable - have you pushed your latest changes?

Reply:
Github is not showing any changes to this button variable - have you pushed your latest changes?

Reply:
I forgot to push after commit, it's there now.